pkg/builder/builder.go

```diff
@@ -161,6 +163,34 @@ func (b Builder) CheckValidation() error {
 		return fmt.Errorf("you cannot use prohibited tags : %s", strings.Join(prohibitedTags, ","))
 	}
 
+	if len(b.Config.Stack) > 0 {
+		hasStack := false
+		for _, stack := range b.Stacks {
+			if stack.Stack == b.Config.Stack {
+				hasStack = true
```

Comment:
Is `break` can be insert to here?

Reply:
Oh, of course it can be. Thanks :)

Reply:
https://github.com/DevopsArtFactory/goployer/pull/85/files#diff-7d55094d2095596359f9af68227cd7cfR171
I fix it here